docs/docs/migration.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: Migration
```

Comment:
Should we call this "Migrating from v1"? We may want to add a "Migrating to v3" in the future.